v0/destinations/iterable/transform.js

```diff
@@ -1,9 +1,17 @@
 const { get } = require("lodash");
```

Comment:
This is not the correct function 
```suggestion
const get = require("get-value");
```

Reply:
Done